components/alert/test/alert-toast.vdiff.js

```diff
@@ -0,0 +1,36 @@
+import '../alert-toast.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const alertWithSubtextAndCloseButton = html`
+	<d2l-alert-toast no-auto-close type="critical" button-text="Do it!" open
+		subtext="I am subtext. I'm here to test the wrapping capabilities of adding subtext to these alerts, as well as other styling issues. Feel free to add to me!">
+		A message.
+	</d2l-alert-toast>
+`;
+
+describe('alert-toast', () => {
+
+	[
+		{ name: 'default', height: 108, template: html`<d2l-alert-toast no-auto-close type="default" open>A default message.</d2l-alert-toast>` },
+		{ name: 'no-close', height: 106, template: html`<d2l-alert-toast no-auto-close type="success" hide-close-button open>A message.</d2l-alert-toast>` },
+		{ name: 'button-close', height: 108, template: html`<d2l-alert-toast no-auto-close type="warning" button-text="Do it!" open>A message.</d2l-alert-toast>` },
+		{ name: 'subtext-button-close', height: 188, template: alertWithSubtextAndCloseButton }
+	].forEach(({ name, template, height }) => {
+		it(name, async() => {
+			await fixture(template, { viewport: { width: 700, height } });
+			await expect(document).to.be.golden();
+		});
+	});
```

Comment:
Ok, I've updated the tests to take a screenshot of the full page and set the height of the viewport to match the old test.  But if we don't care about doing that, I can change it to just "190" for all of them

Reply:
Yeah this seems a little fragile, I'd lean towards just picking a single value that works.